lib/ansible/galaxy/collection/concrete_artifact_manager.py

```diff
@@ -453,6 +453,18 @@ def _extract_collection_from_git(repo_url, coll_ver, b_path):
             proc_err,
         )
 
+    if pathlib.Path(str(b_checkout_path) + "/.gitmodules").is_file():
```

Comment:
There's no point in using pathlib if you don't actually use it:
```suggestion
    if (pathlib.Path(b_checkout_path) / '.gitmodules').is_file():
```